samples/Integration/CMake-fetch-content-app/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
All our other high-level directories are lowercase, change this to lowercase as well:

`samples/integration`, instead of `samples/Integration`.